.github/workflows/release.yml

```diff
@@ -198,4 +215,39 @@ jobs:
           git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
           git config --global user.name "GitHub Action"
           git commit -a -m "Releasing version ${{ needs.precheck.outputs.VERSION }}"
+          git push origin main
+
+  update-bach-info:
+    name: Update bach-info
+    needs: [precheck, release]
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          repository: jreleaser/bach-info
+          ref: 'main'
+          fetch-depth: 0
+          token: ${{ secrets.GIT_ACCESS_TOKEN }}
+
+      - name: Commit
+        run: |
+          VERSION=${{ needs.precheck.outputs.VERSION }}
+          JAR_SIZE=${{ needs.release.outputs.JAR_SIZE }}
+          JAR_CSUM=${{ needs.release.outputs.JAR_CSUM }}
+          DOC_SIZE=${{ needs.release.outputs.DOC_SIZE }}
+          TARGET_FILE=".bach/external-tools/jreleaser@$VERSION.tool-directory.properties"
+          echo "@description JReleaser $VERSION" > $TARGET_FILE
+          echo " " >> $TARGET_FILE
+          echo "jreleaser-tool-provider-$VERSION.jar=\\" >> $TARGET_FILE
+          echo "  https://github.com/jreleaser/jreleaser/releases/download/v$VERSION/jreleaser-tool-provider-$VERSION.jar\\" >> $TARGET_FILE
+          echo "  #SIZE=${JAR_SIZE}&SHA-256=${JAR_CSUM}" >> $TARGET_FILE
+          echo "README.adoc=\\" >> $TARGET_FILE
+          echo "  https://github.com/jreleaser/jreleaser/raw/v$VERSION/README.adoc\\" >> $TARGET_FILE
+          echo "  #SIZE=${DOC_SIZE}" >> $TARGET_FILE
+          git add $TARGET_FILE
```

Comment:
Add an empty `echo ""` to ensure a newline characher at the end of the generated file?

Preventing 🛑 signs like:

![image](https://user-images.githubusercontent.com/2319838/201162169-7702e20b-7187-49d5-b38a-845445cb16cf.png)

Reply:
OK